src/applications/vaos/components/review/AppointmentDate.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import moment from 'moment';
+import { getTimezoneAbbrBySystemId } from '../../utils/timezone';
+
+export default function AppointmentDate(props) {
```

Comment:
Nice work separating everything out into functional components.  Not a requirement, but for functional components it's common practice to destructure the props, similar to how you did for `<ReviewDirectScheduleInfo/>`.  That way you don't have to repeat `props.`. It also gives others a good idea of what data this component expects.  For example:

`export default function AppointmentDate({ dates, systemId })`

This could be done one some of the other components in this PR as well.

Reply:
Thank you!!!